x-pack/packages/observability/logs_overview/kibana.jsonc

```diff
@@ -1,5 +1,9 @@
 {
   "type": "shared-browser",
   "id": "@kbn/observability-logs-overview",
-  "owner": "@elastic/obs-ux-logs-team"
+  "owner": [
+    "@elastic/obs-ux-logs-team"
+  ],
+  "group": "observability",
```

Comment:
@weltenwort @Kerry350 do other solutions depend or meant to depend/embed the logs_overview?

Reply:
Not right now, so it would seem reasonable to me to restrict it to observability for now. :+1: